test-example-diagrams/src/main/resources/simple-service-task/SimpleServiceTask.md

```diff
@@ -0,0 +1,18 @@
+# Timer event
```

Comment:
Here, the header should be **Simple Service Task**

Reply:
Done